WORKSPACE

```diff
@@ -6,6 +6,16 @@ grpc_deps()
 
 grpc_test_only_deps()
 
+# Initialize Google APIs with only C++ and Python targets
+load("@com_google_googleapis//:repository_rules.bzl", "switched_rules_by_language")
+
+switched_rules_by_language(
```

Comment:
How is this dependency used? Is it a test-only dep? If so, this is okay in its current state. If not, then this needs to go in the `grpc_extra_deps` macro. Otherwise, we'll break repos pulling us in through Bazel.

Reply:
Nice idea. Added to `grpc_extra_deps`, so people should be able to get it easily.

This dependency is used by one public C++ library target `grpcpp_csds` and tests. The Python targets will hopefully be needed by the interop test driver, and in near future, I might use it to port CSDS to Python.enabled Python targets here.

Reply:
Just a note, this caused a quite painful debugging after upgrading to gRPC 1.37.0 (first release with this commit) - I was calling `switched_rules_by_language` after `grpc_extra_deps` in my workspace, which ended up being ignored - so I only got C++ and Python, while I was also interested in Java support. `switched_rules_by_language` just silently ignores every call after the first, apparently.

Where this might get really problematic is the other way around - what happens if I call `switched_rules_by_language` with e.g. _just_ Java support before calling `grpc_extra_deps`? I wonder if we can do something not involving having `switched_rules_by_language` in there...

Reply:
@Kernald I wasn't aware that `switched_rules_by_language` did this. Yes, this sounds very problematic. You could easily get into a diamond dependency situation where you depend on repo A and repo B, both using different `switched_rules_by_language` calls and with one always failing depending on the order in which you pull them in.

@lidizheng Creating [an issue](https://github.com/grpc/grpc/issues/25931) to track this.